monad-logger-extras.cabal

```diff
@@ -34,7 +34,7 @@ library
     , base           >=4.12   && <5
     , bytestring     >=0.10   && <0.12
     , hsyslog        >=5      && <5.1
-    , monad-logger   >=0.3.36 && <0.4
+    , monad-logger   >=0.3.40 && <0.4
```

Comment:
Perhaps it's a good opportunity to be PVP compliant and add upper bounds on the minor version, re:
> 6. Client defines orphan instance. If a package defines an orphan instance, it MUST depend on the minor version of the packages that define the data type and the type class to be backwards compatible. For example, build-depends: mypkg >= 2.1.1 && < 2.1.2.